dash_bio/component_factory/_volcano.py

```diff
@@ -274,9 +290,11 @@ def __init__(
 
     def figure(
             self,
-            title='Volcano Plot',
+            title=None,
             xlabel=None,
             ylabel='-log10(p)',
+            xaxis=None,
+            yaxis=None,
```

Comment:
Is there any difference between explicitly including these to pass to `figure.update_xaxes(**xaxis)`, and omitting them so they get picked up by `layout.update(**kwargs)`? I suppose if we were to have multiple x or y axes, the former would update all of them, the latter would only update the first pair. But I don't believe we do have multiple axes, and anyway if we did I imagine you'd want the ability to update them separately.

Reply:
I was worried that key-worded arguments wouldn't be unpacked down to `xaxes` without overwriting the defaults, but after seeing your comment about the deep merge and testing it out, there is no difference. Fixed in 56948c3.